pycoal/tests/mineral_test.py

```diff
@@ -81,7 +82,36 @@ def test_classify_image():
         # verify that every pixel has the same classification
         assert numpy.array_equal(expected.asarray(), actual.asarray())
 
+# verify that classified images have valid classifications when using config file
```

Comment:
This is looking better. Please go ahead and implement the **config.ini** file and then work on the configuration management logic. This will involve 1) loading the file and also 2) augmenting the various [__init__](https://github.com/capstone-coal/pycoal/blob/master/pycoal/mineral.py#L288) functions in mineral, mining and environment.py to accommodate config.ini.

Reply:
I've updated the mineral.py file to support setting the algorithm (SAM or avngDNN) and implementation via the config file. Now all that is needed is to add the different parallelized algorithms as functions to call into.